plugin.video.cbcolympics/addon.xml

```diff
@@ -1,21 +1,27 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.cbcolympics"
        name="CBC Olympics"
-       version="1.0.0"
+       version="1.0.1"
        provider-name="fourpointsix">
   <requires>
-    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="xbmc.python" version="2.24.0"/>
```

Comment:
Please be aware bumping the version down will only allow manual installation in Kodi 16. v.16 users won't see the plugin in Kodis repository.

Reply:
Yes, I'm OK with that.